xarray/tests/test_dask.py

```diff
@@ -149,8 +149,8 @@ def test_binary_op(self):
     def test_repr(self):
         expected = dedent(
             """\
-        <xarray.Variable (x: 4, y: 6)>
-        dask.array<shape=(4, 6), dtype=float64, chunksize=(2, 2)>"""
+            <xarray.Variable (x: 4, y: 6)>
+            dask.array<array, shape=(4, 6), dtype=float64, chunksize=(2, 2)>"""
```

Comment:
Was it a conscious decision to add the array name? Shouldn't it be coherent with inline_dask_repr?

Reply:
In Variable and DataArray, we are now using the repr of the underlying data directly (which includes `name` for dask arrays), rather than additional processing with xarray. That seemed like an improvement to me, given that we aren't as strictly space constrainted as we are in the inline Dataset repr.